mods/win7-alttab-loader.wh.cpp

```diff
@@ -0,0 +1,668 @@
+// ==WindhawkMod==
+// @id              win7-alttab-loader
+// @name            Windows 7 Alt+Tab Loader
+// @description     Loads Windows 7 Alt+Tab on Windows 10.
+// @version         1.0
+// @author          ephemeralViolette, aubymori
+// @include         explorer.exe
+// @compilerOptions -lole32 -luuid -ldwmapi -luxtheme -ldbghelp

```

Comment:
If this mod is x64-only, you can add `@architecture x86-64` to avoid compiling a 32-bit version.